CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## v4.4.2 - 2025-01-13 - 7ad7a106b
 
+- Fix #2053: Update RDS policy to allow createDBSnapshot during destroy
+- Feat #1897: Upgrade OS version to CentOS Stream 9
```

Comment:
Can these 2 commit messages be moved into `Unreleased` section please?

Reply:
Fixed